### PR TITLE
rake task for dumping raw data for plots

### PR DESCRIPTION
**Pull request by gedankenstuecke**

### CODE REVIEW

**Comment by tsujigiri** [comment]
Can't the plot scripts connect to the database directly?

**Comment by gedankenstuecke** [comment]
Probably could somehow, but offering those files wouldn't be too bad either, would it? And [I also did the R scripts + Docker](https://github.com/openSNP/growthplots). Does that make sense?

**Comment by tsujigiri** [comment]
OK, this will do as well.

**Comment by gedankenstuecke** [comment]
Yay! :) I had some comments to your comments on the whole Dockerfile issue though :smile:

File: lib/tasks/dump_numbers.rake [hunk 1]
```diff
@@ -0,0 +1,42 @@
+namespace :numbers do
+  desc "dump numbers for R plots"
+  task :dump => :environment do
+    # check whether directory for dump exists or create
+    Dir.mkdir("#{Rails.root}/public/data/plot_data/") unless File.exists?("#{Rails.root}/public/data/plot_data/")    
+    # start with getting users
+    counter = 0
+    File.open("#{Rails.root}/public/data/plot_data/number_users.csv","w"){ |file|
+      User.find_each do |u|
```
Comment (tsujigiri) [inline lib/tasks/dump_numbers.rake]: Instead of `counter` you can do `User.find_each.with_index do |u, i|` and `file.puts("#{i + 1}\t#{u.created_at}")`

**Comment by gedankenstuecke** [comment]
looks good otherwise?

**Comment by tsujigiri** [comment]
:cake:

**Pull request merged**